requirements.txt

```diff
@@ -3,4 +3,5 @@ librosa
 numpy
 tokenizers>=0.19.0
 torch
-transformers>=4.40.0
\ No newline at end of file
+transformers>=4.40.0
+git+https://github.com/modelscope/FunCodec@master
```

Comment:
There is a PyPI package for funcodec [https://pypi.org/project/funcodec/](https://pypi.org/project/funcodec/). Can we use this instead of installing from git? I'm not sure if its actively maintained or not, considering the Readme still shows installation from source, but I tried installing the PyPI package and it seems to work with your code.